From e0c57944e332c88fbbf2b8304e007a4c06312597 Mon Sep 17 00:00:00 2001
From: Tom Cunningham <tcunning@redhat.com>
Date: Fri, 27 Oct 2023 09:48:45 -0400
Subject: [PATCH] Add snyk configuration file excluding tests (#2335)

---
 .gitignore | 1 +
 .snyk      | 5 +++++
 2 files changed, 6 insertions(+)
 create mode 100644 .snyk

diff --git a/.gitignore b/.gitignore
index 353435dcccfd..d631a668da7b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -3,6 +3,7 @@ target
 *.ipr
 *.iws
 .idea
+.dccache
 .DS_Store
 .classpath
 .ekstazi
diff --git a/.snyk b/.snyk
new file mode 100644
index 000000000000..3764ff72ca01
--- /dev/null
+++ b/.snyk
@@ -0,0 +1,5 @@
+exclude:
+    global:
+        - "*IT.java"
+        - "*Test.java"
+        - "*src/test/*"
-- 
2.46.0

